README.md

```diff
@@ -12,3 +12,7 @@ $ google-chrome-stable --remote-debugging-port=9899
 In a python console, you can connect to the remote debugging port and enable the Page domain.
 
 > devtools_client = ChromeInterface(9899, domains={"Page": {}})
+
+In a python console, you can connect to the remote debugging port and enable the Page domain.
+
+>> devtools_client = ChromeInterface(9899, domains={"Page": {}})
```

Comment:
You have already completed this work? This is the same line as the one above.

Reply:
Okay, So I'm closing this pull request.